cmd/networkagent/main.go

```diff
@@ -0,0 +1,453 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math/rand"
+	"net"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+	"github.com/Microsoft/hcsshim/cmd/ncproxy/ncproxygrpc"
+	"github.com/Microsoft/hcsshim/cmd/ncproxy/nodenetsvc"
+	"github.com/Microsoft/hcsshim/hcn"
+	"github.com/Microsoft/hcsshim/internal/log"
+	"google.golang.org/grpc"
+)
+
+// This is a barebones example of an implementation of the network
+// config agent service that ncproxy talks to. This is solely used to test and
+// will be removed.
```

Comment:
I think the last statement can be removed now, this was probably a relic of my old one 😆